python/pfnet/.gitignore

```diff
@@ -0,0 +1,2 @@
+/libpfnet.0.dylib
+/libpfnet.dylib
```

Comment:
Probably better to use wildcards to ignore any dylibs and libs for macos
*.dylib
*.lib